ui/uihandler.go

```diff
@@ -843,9 +843,12 @@ func Serve(baseurl string, port int, debugpath, uipath, staticRsrcDir string, da
 		uiPath += "/"
 	}
 	debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
+	httpPort = fnet.NormalizePort(port)
+	if strings.HasPrefix(httpPort, ":") {
+		httpPort = "localhost" + httpPort
```

Comment:
that global variable is used in the templates, I don't think you can change it that way without breaking the UI

Reply:
My bad. I will create a local var and use that instead of httpPort.

On lines 847-848 I am add the string "localhost" if the port starts with ":". After further thought, it should be "0.0.0.0" instead of "localhost". This is because the http.Serve function is binding `port` to all addresses when port is not specified as `ip:port`.

Reply:
can we get the actual address from the listener?